navigation.yaml

```
@@ -327,6 +327,8 @@ download:
         - title: UP Squared IoT Grove
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
+          title: Installation medias
```

This presumably needs a dash to denote that it's a new item in the list:

```suggestion

         - title: Installation media
```